templates/RA.html.tmpl

```diff
@@ -436,10 +440,18 @@
     <div class="page">
         <h2 id="section_6"><span>6</span>Definitions and Abbreviations</h2>
         <h3 id="section_6_1"><span>6.1</span>Definitions</h3>
-		<p>{{{data.sections.sec6s1.content}}}</p>
+        <table>
+            <tr>
+                <td class="content-wrappable">{{{data.sections.sec6s1.content}}}</td>
+            </tr>
+        </table>
 
 		<h3 id="section_6_2"><span>6.2</span>Abbreviations</h3>
-		<p>{{{data.sections.sec6s2.content}}}</p>
```

Comment:
not sure git is indenting this line, but the table that replaces the `<p>` is indented correctly